arbitrator/prover/src/machine.rs

```diff
@@ -2483,9 +2546,50 @@ impl Machine {
                 hash_stack_with_heights(frames, &heights, concat!($prefix, " stack:"))
             }};
         }
-        let (frame_stack, frames) =
-            compute!(|x| x.frame_stack, self.get_frame_stack(), "Stack frame");
-        let (value_stack, values) = compute!(|x| x.value_stack, self.get_data_stack(), "Value");
+        // compute_multistack returns the hash of multistacks as follows:
+        // Keccak(
+        //      "multistack: " 
+        //      + hash_stack(first_stack)
+        //      + hash_stack(last_stack)
+        //      + Keccak("cothread: " + 2nd_stack+Keccak("cothread" + 3drd_stack + ...)
```

Comment:
code does "cothread:" (no space)

Reply:
Done. Dropped it from multistack for consistency as well.